types/prosemirror-view/package.json

```diff
@@ -0,0 +1,11 @@
+{	
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/prosemirror-view/package.json;
  Line: 1;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
Where do you see non-spaces being used for new or changed types?

Reply:
That's actually a bot.  Bizarrely, it looks like there are trailing tabs on the lines of this file.